assets/css/input-number.css

```diff
@@ -0,0 +1,170 @@
+.rc-input-number {
```

Comment:
I would appreciate if this changes were included in a seperated PR, to find them easily when you look them in PRs.
But well, let's move forward